### DISCUSSION TITLE
Only getting advert videos when using VDH on Rumble

### DISCUSSION THREAD

**Discussion by feather42** [aclap-dev/video-downloadhelper#39] [Help and Q&A] [answered]
### Your question

I am using VDH v8.1.4.0 with companion app v2.0.10 installed on fully patched Windows 11 using Firefox browser
If I try to download a video from Rumble and that video has adverts on it then the only video hits I get to DL are the adverts there is no item to DL the main video. As an example try this video its got ad's on it https://rumble.com/v4b5ral-a-motive-andrew-tate-on-suffering.htm
If I go to DL a video that has no adverts on it then there is no problem I get the actual video.
I dont DL from Rumble often and most videos there dont run adverts so have no idea when this started happening.

As a work around if I watch part of the video, goto another video then go back to the original one from my "continue watching" section on the rumble homepage I can get the actual video I assume this is because it jumps back in at the point you previously stopped at and doesnt play any adverts.

Is anybody else having this problem?
Can the dev's look into this at some point pls.

Thanks.

### Addon & CoApp version and details

version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-GB
license: accepted
key (only first 16 fisrt characters): a4d1c1c789864d12
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\ms",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment, marked as answer]
Mind trying version 8.2 Beta? https://downloadhelper.net/beta

**Reply by feather42** [reply]
Installed the Beta and tested on another video with adverts on Rumble and it works fine there is now an option to DL the main video as well as the adverts. Thanks..
